embedded/common/modules/sensor-drivers/Driver_Sensor.h

```diff
@@ -0,0 +1,98 @@
+/* Open Sensor Platform Project
```

Comment:
Why burry this critical shared header file so deep in the heirarchy? I would expect to see these in a common/include folder or something at a similar high level